src/nvim/edit.c

```diff
@@ -5899,7 +5899,7 @@ comp_textwidth (
       textwidth -= 1;
     }
     textwidth -= curwin->w_p_fdc;
-    textwidth -= win_signcol_count(curwin);
+    textwidth -= win_signcol_width(curwin);
```

Comment:
Should/could address the buffer value directly probably for performance reasons.